services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/converter/OfMeterStatsConverter.java

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.floodlight.converter;
+
+import static java.util.stream.Collectors.toList;
+
+import org.openkilda.messaging.info.stats.MeterStatsData;
+import org.openkilda.messaging.info.stats.MeterStatsEntry;
+import org.openkilda.messaging.info.stats.MeterStatsReply;
+import org.openkilda.model.SwitchId;
+
+import org.projectfloodlight.openflow.protocol.OFMeterStats;
+import org.projectfloodlight.openflow.protocol.OFMeterStatsReply;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public final class OfMeterStatsConverter {
+    private static final Logger logger = LoggerFactory.getLogger(OfMeterStatsConverter.class);
+
+    private OfMeterStatsConverter() {}
```

Comment:
There's a good practice to throw a UnsupportedOperationException in private constructors of utility classes.

Reply:
fixed